exercises/numbers/isPositive/isPositive.js

```diff
@@ -15,6 +15,8 @@ if (require.main === module) {
   console.log(isPositive(1) === true);
   console.log(isPositive(-1) === false);
   console.log(isPositive(-100.201) === false);
+  // console.loge(())
+    // How do I make it so that user input can also be determined as positive or not
```

Comment:
Look into nodejs arguments

Reply:
If you want user input, check out the examples in: https://github.com/jfarmer/examples-javascript